### DISCUSSION TITLE
Confusing description of spacing utilities notation

### DISCUSSION THREAD

**Discussion by Stella521** [twbs/bootstrap#39643] [Q&A] [answered]
Hi everyont, I'm new to front end developing and Bootstrap. I'm a bit confused about some descriptions in the Notation documentation of spacing uilities. I'm using the link here (https://getbootstrap.com/docs/5.3/utilities/spacing/). It says "Spacing utilities that apply to all breakpoints, from xs to xxl, have no breakpoint abbreviation in them. This is because those classes are applied from min-width: 0 and up, and thus are not bound by a media query. The remaining breakpoints, however, do include a breakpoint abbreviation." Could anyone please give me some guidance on what does "the remaining breakpoints" mean? And in the next paragraph, it says "The classes are named using the format {property}{sides}-{size} for xs and {property}{sides}-{breakpoint}-{size} for sm, md, lg, xl, and xxl." . Should I understand the classes using different formats for xs and (sm, md, lg, xl, and xxl) are "the remaining breakpoints"?

**Comment by julien-deramond** [comment, marked as answer]
Hey @Stella521 

> Should I understand the classes using different formats for xs and (sm, md, lg, xl, and xxl) are "the remaining breakpoints"?

The "remaining breakpoints" are all breakpoints except `xs`, so `sm`, `md`, `lg`, `xl` and `xxl`.

For example:
* `mt-0` (that corresponds to `xs`) has no abbreviation and is defined without any CSS media
* `mt-sm-0` is defined from `sm` with the following CSS media: `@media (min-width: 576px)`
* `mt-md-0` is defined from `md` with the following CSS media: `@media (min-width: 768px)`
* `mt-lg-0` is defined from `lg` with the following CSS media: `@media (min-width: 992px)`
* `mt-xl-0` is defined from `xl` with the following CSS media: `@media (min-width: 1200px)`
* `mt-xl-0` is defined from `xl` with the following CSS media: `@media (min-width: 1400px)`

**Reply by Stella521** [reply]
Hi Julien, thank you very much for the clarification, that indeed makes sense and I could understand completely now. Just wondering, in this case, woud it be better if the description is "Spacing utilities of xs" instead of simply saying "spacing utilities" in the beginning of the first paragraph? Because in my personal opinion, it is a bit difficult to understand the remaining brakpoints are all breakpoints expect xs if xs is not mentioned specifically  in the first paragraph.

**Reply by julien-deramond** [reply]
Thanks for the suggestion @Stella521, I'll add that to my TODO list to see how it could be improved. I agree that, as a non-native English speakers, I struggled to understand the "remaining breakpoints" even by knowing how it works.
